tests/parametric/test_otel_span_methods.py

```diff
@@ -286,3 +286,774 @@ def test_otel_get_span_context(self, test_agent, test_library):
                     assert context.get("trace_id") == parent.span_context().get("trace_id")
                     assert context.get("span_id") == "{:016x}".format(span.span_id)
                     assert context.get("trace_flags") == "01"
+
+    @missing_feature(context.library == "go", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @irrelevant(context.library == "cpp", reason="library does not implement OpenTelemetry")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
```

Comment:
If you are going to mark as "missing_feature" all method of the class for one language, you should remove these class decorator and use manifest files instead. Check manifests folder and doc: https://github.com/DataDog/system-tests/blob/main/docs/RFCs/manifest.md

Reply:
Thanks @robertomonteromiguel I didn't know about the manifest files - I added to `cpp`, `ruby`, and `php`'s in https://github.com/DataDog/system-tests/pull/1655/commits/0a1f682d78999a11c3dce37dafe556b1fa1731ad